packages/manager/modules/pci/src/projects/project/data-processing/submit-job/spark-sizing/spark-sizing.html

```diff
@@ -166,21 +177,30 @@
                 <oui-numeric
                     name="worker-memory-overhead"
                     data-model="$ctrl.state.workerMemoryOverheadMb"
-                    data-min="$ctrl.minMemoryOverheadMb"
-                    data-max="30000"
+                    data-min="$ctrl.jobParameters.executor_memory_overhead.validator.min"
+                    data-max="$ctrl.jobParameters.executor_memory_overhead.validator.max"
+                    data-disabled="$ctrl.state.advancedSizingWorkerMemOverheadAuto"
                 >
                 </oui-numeric>
             </div>
-            <span class="sizing__advanced-units">MB</span>
+            <span class="sizing__advanced-units">MiB</span>
+            <oui-checkbox
+                class="sizing__advanced-units-checkbox"
+                name="worker-memory-overhead-auto"
+                data-model="$ctrl.state.advancedSizingWorkerMemOverheadAuto"
+                data-on-change="$ctrl.onChangeAdvancedSizingWorkerMemOverheadHandler(modelValue)"
+            >
+                <span>Automatically compute</span>
```

Comment:
Could you please extract it into a translation file?

Reply:
Commit amended with correction